### ISSUE TITLE
Instance-reset doesn't work?

### ISSUE THREAD

**Issue by deadcorpseeater** [ZhengPeiRu21/mod-individual-progression#398]
there is no instance-reset npc after turning the module on in game
https://github.com/ZhengPeiRu21/mod-individual-progression/releases/tag/20240821

**Comment by Grimfeather** [comment]
https://github.com/azerothcore/mod-instance-reset

Under "installation" it says:
- Add the npc via script or command wherever you like.
- npc add 300000 or npc add temp 300000 (If you want to add it temporarily)

so by default there is no npc yet. you have to add them yourself to locations you find suitable.
gm account required.

**Comment by deadcorpseeater** [comment]
> https://github.com/azerothcore/mod-instance-reset
> 
> Under "installation" it says:
> 
> * Add the npc via script or command wherever you like.
> * npc add 300000 or npc add temp 300000 (If you want to add it temporarily)
> 
> so by default there is no npc yet. you have to add them yourself to locations you find suitable. gm account required.

I found this, but there's no such a creature in this serverpack. I used instance-reset before, so I was really surprised there is no "Cromi"

![image](https://github.com/user-attachments/assets/c1437044-70a6-4b6d-ada0-164372a976bb)

**Comment by Grimfeather** [comment]
I downloaded the preview repack to check.
Looks like the SQL included with the instance reset module wasn't run on the preview repack.


SET
@Entry      := 300000,
@Model      := 24877,
@Name       := "Cromi",
@Title      := "Instance Reset";

DELETE FROM `creature_template` WHERE `entry` = @Entry;

INSERT INTO `creature_template` (`entry`, `name`, `subname`, `IconName`, `gossip_menu_id`, `minlevel`, `maxlevel`, `faction`, `npcflag`, `speed_walk`, `speed_run`, `scale`, `rank`, `unit_class`, `unit_flags`, `type`, `type_flags`, `RegenHealth`, `flags_extra`, `ScriptName`) VALUES
(@Entry, @Name, @Title, "Speak", 0, 80, 80, 35, 0, 1, 1.14286, 1, 0, 1, 2, 7, 138936390, 1, 2, "instance_reset");

DELETE FROM `creature_template_model` where creatureid=@Entry;

INSERT INTO `creature_template_model` (CreatureID, Idx, CreatureDisplayID, DisplayScale, Probability, VerifiedBuild) VALUES
(@Entry, '0', @Model, '1', '1', NULL);

UPDATE `creature_template` SET `npcflag`=`npcflag`|1, `flags_extra`=`flags_extra`|16777216 WHERE `entry`=@Entry;

DELETE FROM `creature_template_locale` WHERE `entry`=@Entry;

INSERT INTO `creature_template_locale` (`entry`, `locale`, `Name`, `Title`, `VerifiedBuild`) VALUES
(@Entry, 'esES', @Name, 'Reinicio de instancias', 0),
(@Entry, 'esMX', @Name, 'Reinicio de instancias', 0),
(@Entry, 'frFR', @Name, 'Redémarrage des instances', 0),
(@Entry, 'koKR', @Name, '인스턴스 재시작', 0),
(@Entry, 'deDE', @Name, 'Neustart der Instanz', 0),
(@Entry, 'zhCN', @Name, '重新启动实例', 0),
(@Entry, 'zhTW', @Name, '重置副本', 0),
(@entry, 'ruRU', @name, 'Перезапуск подземелий', 0);

**Comment by deadcorpseeater** [comment]
> I downloaded the preview repack to check. Looks like the SQL file included with the instance reset module wasn't run on the preview repack. The Cromi npc doesn't exist yet in the database. It needs to be created still.
> 
> I suggest you download the instance reset module yourself. You'll find the SQL in: mod-instance-reset-master.zip\mod-instance-reset-master\data\sql\db-world\base
> 
> it's called: mod_instance_reset_2024_03_14.sql
> 
> if you have HeidiSQL installed you can simply double click the sql file. After you logged into the database you have to run the SQL with F9. you log into the database with username root no password.

thank you for help! will do it later after vacation

**Comment by deadcorpseeater** [comment]
> I downloaded the preview repack to check. Looks like the SQL file included with the instance reset module wasn't run on the preview repack. The Cromi npc doesn't exist yet in the database. It needs to be created still.
> 
> I suggest you download the instance reset module yourself. You'll find the SQL in: mod-instance-reset-master.zip\mod-instance-reset-master\data\sql\db-world\base
> 
> it's called: mod_instance_reset_2024_03_14.sql
> 
> if you have HeidiSQL installed you can simply double click the sql file. After you logged into the database you have to run the SQL with F9. you log into the database with username root no password.

done it using HeidiSQL, NPC appears but has no options

**Comment by Grimfeather** [comment]
Maybe the module wasn't added correctly when this preview repack was compiled. 
I recommend you use a macro to reset instances instead.
.instance unbind all
or check out:  .instance unbind 
for more options.

It's not important. It doesn't effect the individual progression module.

**Comment by deadcorpseeater** [comment]
> Maybe the module wasn't added correctly when this preview repack was compiled. I recommend you use a macro to reset instances instead. .instance unbind all or check out: .instance unbind for more options.
> 
> It's not important. It doesn't effect the individual progression module.

does it work with raid instances?

**Comment by Grimfeather** [comment]
yes, a GM account is required though to use the .instance command.

**Comment by deadcorpseeater** [comment]
> yes, a GM account is required though to use the .instance command.

ty, you saved me)

**Comment by Grimfeather** [comment]
If you want a more up to date version of the preview build for this module, then you can find it here:
https://github.com/Grimfeather/mod-individual-progression/releases/tag/2024-11-15

I created it today, it has all the latest updates.
